mesonpy/__init__.py

```diff
@@ -854,6 +855,17 @@ def sdist(self, directory: Path) -> pathlib.Path:
 
         with tarfile.open(meson_dist_path, 'r:gz') as meson_dist, mesonpy._util.create_targz(sdist_path) as sdist:
             for member in meson_dist.getmembers():
+                # Wheels do not support links, though sdist tarballs could. For portability, reduce these to regular files.
+                if member.islnk() or member.issym():
+                    # Symlinks are relative to member directory, but hard links are relative to tarball root.
+                    path = member.name.rsplit('/', 1)[0] + '/' if member.issym() else ''
+                    orig = meson_dist.getmember(path + member.linkname)
```

Comment:
Are meson or git enforcing that symlinks do not point to location outside the tarball or outside the repository?

Reply:
git does not appear to enforce any rules about committed symlinks.